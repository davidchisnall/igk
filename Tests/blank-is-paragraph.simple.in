
A paragraph
of text.

Another paragraph.
This one has two sentences.


