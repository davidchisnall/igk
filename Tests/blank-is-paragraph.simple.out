\p{
A paragraph
of text.}\p{Another paragraph.
This one has two sentences.}